src/index.js

```diff
@@ -1,7 +1,62 @@
 import './style.css';
-import { scores } from './api.js';
-import { fillScores } from './DOM.js';
+import { toast } from 'wc-toast';
+import { fetch } from './api.js';
+import { fillScores, getHash, addScore } from './DOM.js';
+
+async function refreshDom() {
+  await getHash
+    .then(
+      (resolved) => {
+        fetch(resolved)
+          .then(
+            (resolved) => {
+              console.log('resolved:', resolved);
+
+              fillScores(resolved);
+            },
+            (rejected) => {
+              console.log(rejected);
+              toast(rejected);
+            },
+          )
+          .catch((err) => {
+            console.log(err);
+            toast(err);
+          });
+      },
+      (rejected) => {
+        console.log(rejected);
+        toast(rejected);
+      },
+    )
+    .catch((err) => toast(err));
+}
```

Comment:
Well done for implementing almost all the project requirements to near-perfection, but kindly use the arrow functions instead of the function key word for this particular part of your code :wink: